src/components/Stepper/src/Stepper.vue

```diff
@@ -124,22 +155,57 @@ export default {
 				return;
 			}
 			const incrementBy = 1;
-			/**
-			 * updated stepper value
-			 * @property {number}
-			 */
-			this.$emit('stepper:update', this.value + incrementBy);
+			this.emitUpdate(this.value + incrementBy);
 		},
 		decrement() {
 			if (!Number.isNaN(this.minVal) && this.value <= this.minVal) {
 				return;
 			}
 			const decrementBy = 1;
+			this.emitUpdate(this.value - decrementBy);
+		},
+
+		triggerManualInput() {
+			this.manualValue = this.value;
+			this.isSettingManualValue = true;
+			this.$nextTick(() => {
+				this.$refs.manualInput.focus();
+				this.$refs.manualInput.select();
+			});
+		},
+
+		commitManualValue(event) {
+			event?.preventDefault?.();
+			event?.stopPropagation?.();
+
+			const newValue = Number.parseInt(this.manualValue, 10);
+			this.isSettingManualValue = false;
+
+			if (Number.isNaN(newValue) || newValue === null || newValue === undefined) {
```

Comment:
Not sure if / what we could do about this, but there's a few characters that get through:

![CleanShot 2022-05-06 at 12 27 38](https://user-images.githubusercontent.com/129122/167204817-6a726989-968a-44f3-a2ef-b31405a83d97.png)

Reply:
`e` is a valid number value, used for writing exponents in scientific notation, e.g. `1e2` = 100. A small issue is that the `Number.parseInt` doesn't correctly handle values with `.` or `e` characters in them :/

![image](https://user-images.githubusercontent.com/7769424/167409942-43ed005f-105f-45c1-84ae-6b7ad473e053.png)